resources/keywords/dbaas_policy.resource

```diff
@@ -66,5 +76,40 @@ Create Custom DBaaSPolicy
     Input Text    ${txtBxConnNS}    *
     Scroll Element Into View    ${btnCreateEnabled}
     Click Element    ${btnCreateEnabled}
-    ${lnkDBaaSPolicyUpd} =    Replace String    ${lnkDBaaSPolicyName}    <<policyname>>   ${policyName}
-    Wait Until Page Contains Element    ${lnkDBaaSPolicyUpd}    timeout=10s
+
+User Creates DBaaSPolicy For ${namespace} From Invalid Namespace Alert on Admin View As ${DBaaSPersona}
```

Comment:
This is a case where a namespace does not have a policy?

Reply:
Yes @ikanias, And we would be creating DBaaSPolicy from the alert screen which shows Namespace is invalid as DBaaSPolicy has not been added to it.